### DISCUSSION TITLE
Atom editor in browser

### DISCUSSION THREAD

**Discussion by Pranomvignesh** [atom/atom#22137] [Ideas]
Is there any project on going in atom, to make it work in the browser like how vscode is having monaco editor ?
Please update me if I am unaware of an existing product

**Comment by ThatXliner** [comment]
I think this is relevant: https://flight-manual.atom.io/faq/sections/atom-in-the-cloud/

**Reply by Pranomvignesh** [reply]
Thanks @ThatXliner . Will look into it

**Comment by datavectors** [comment]
See [this thread](https://github.com/atom/atom/discussions/22398) on same topic.
